core/src/test/java/org/apache/spark/shuffle/sort/UnsafeShuffleWriterSuite.java

```diff
@@ -40,9 +41,11 @@
 import org.mockito.stubbing.Answer;
 
 import org.apache.spark.HashPartitioner;
+import org.apache.spark.SecurityManager;
 import org.apache.spark.ShuffleDependency;
 import org.apache.spark.SparkConf;
 import org.apache.spark.TaskContext;
+import org.apache.spark.deploy.SparkHadoopUtil;
```

Comment:
other than `CryptoStreamUtils`, the other added imports look unused.  Also looks like you can eliminate `AbstractFunction1` and `ByteStreams` since you are no longer using them.